dpc/src/program/noop_program.rs

```diff
@@ -74,33 +71,10 @@ impl<C: Parameters> ProgramScheme for NoopProgram<C> {
         })
     }
 
-    // TODO (howardwu): Why are we not preparing the VK here?
     /// Loads an instance of the noop program.
     fn load() -> Result<Self, ProgramError> {
-        let proving_key: <Self::ProofSystem as SNARK>::ProvingKey = match Network::from_id(C::NETWORK_ID) {
-            Network::Testnet1 => FromBytes::read_le(
-                snarkvm_parameters::testnet1::NoopProgramSNARKPKParameters::load_bytes()?.as_slice(),
-            )?,
-            Network::Testnet2 => FromBytes::read_le(
-                snarkvm_parameters::testnet2::NoopProgramSNARKPKParameters::load_bytes()?.as_slice(),
-            )?,
-            _ => {
-                unimplemented!()
-            }
-        };
-
-        let verifying_key = match Network::from_id(C::NETWORK_ID) {
-            Network::Testnet1 => <Self::ProofSystem as SNARK>::VerifyingKey::read_le(
-                snarkvm_parameters::testnet1::NoopProgramSNARKVKParameters::load_bytes()?.as_slice(),
-            )?,
-            Network::Testnet2 => <Self::ProofSystem as SNARK>::VerifyingKey::read_le(
-                snarkvm_parameters::testnet2::NoopProgramSNARKVKParameters::load_bytes()?.as_slice(),
-            )?,
-            _ => {
-                unimplemented!()
-            }
-        };
-
+        let proving_key = C::noop_program_proving_key().clone();
+        let verifying_key = C::noop_program_verifying_key().clone();
```

Comment:
nit: `NoopProgram` can probably just contain those 2 `&'static`s without the need to ever clone them.

Reply:
Discussed offline, Derivative doesn't cooperate well with this change.